.evergreen.yml

```diff
@@ -578,6 +614,215 @@ tasks:
     commands:
       - func: "clone"
       - func: "set up notary client credentials"
+      - func: "install curator"
+      - func: "push"
+        vars:
+          edition: org
+          distro: debian10
+          ext: deb
+  ################################################################
+  # Stable Publish
+  # RPM Distros
+  - name: push_rhel70_enterprise_44
+    <<: &default_push_stable_def
+      patch_only: true
+      stepback: false
+      tags: ["push_stable"]
+      depends_on:
+        - name: compile
+          variant: "code_health"
+        - name: release
+          variant: "release_publish"
```

Comment:
We only one to run this after our goreleaser task, goreleaser needs to have a valid tag at the time of running so we can only run this with a patch as evergreen does not support events on tags yet
This adds a set of tasks for our release candidate, but we may need to add another set of tasks or variant if we need to publish to the 4.2 repo as well